### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Habtisho** [community/community#48952] [New to GitHub]
### Select Topic Area

Question

### Body

I am a beginner please help me how I develop my software experience?

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @Habtisho  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
